eng/deploy.yaml

```diff
@@ -16,6 +16,7 @@ parameters:
   # dn-bot-dnceng-build-rw-code-rw-release-rw
   # maestro-scenario-test-github-token
   # dotnet-build-bot-dotnet-eng-status-token
+  # secret-manager-scenario-tests-auth-connection-string
```

Comment:
This name doesn't match the name that is used below.

Reply:
I renamed it at some point and forgot to rename it in this comment. Thank you for spotting it, it's updated now.